pkgs/by-name/gl/glitchtip/package.nix

```diff
@@ -0,0 +1,131 @@
+{
+  lib,
+  python313,
+  fetchFromGitLab,
+  callPackage,
+  stdenv,
+  makeWrapper,
+  nixosTests,
+}:
+
+let
+  python = python313.override {
+    packageOverrides = final: prev: {
+      django = final.django_5;
+      django-extensions = prev.django-extensions.overridePythonAttrs { doCheck = false; };
+    };
+  };
+
+  pythonPackages =
+    with python.pkgs;
+    [
+      aiohttp
+      anonymizeip
+      boto3
+      brotli
+      celery
+      celery-batches
+      dj-stripe
+      django
+      django-allauth
+      django-anymail
+      django-cors-headers
+      django-csp
+      django-environ
+      django-extensions
+      django-import-export
+      django-ipware
+      django-ninja
+      django-organizations
+      django-prometheus
+      django-redis
+      django-rest-mfa
+      django-sql-utils
+      django-storages
+      google-cloud-logging
+      gunicorn
+      orjson
+      psycopg
+      pydantic
+      sentry-sdk
+      symbolic
+      user-agents
+      uvicorn
+      uwsgi-chunked
+      whitenoise
+    ]
+    ++ celery.optional-dependencies.redis
+    ++ django-allauth.optional-dependencies.mfa
+    ++ django-allauth.optional-dependencies.socialaccount
+    ++ django-redis.optional-dependencies.hiredis
+    ++ django-storages.optional-dependencies.boto3
+    ++ django-storages.optional-dependencies.azure
+    ++ django-storages.optional-dependencies.google
+    ++ psycopg.optional-dependencies.c
+    ++ psycopg.optional-dependencies.pool
+    ++ pydantic.optional-dependencies.email;
+
+  frontend = callPackage ./frontend.nix { };
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "glitchtip";
+  version = "4.2.5";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "glitchtip";
+    repo = "glitchtip-backend";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-OTf2rvx+ONnB7pLB7rinztXL7l2eZfIuI7PosCXaOH8=";
+  };
+
+  propagatedBuildInputs = pythonPackages;
```

Comment:
We want to use use dependencies and build-system here, too

Reply:
This is not a buildPython... so these parameters doesn't exist.

Reply:
oh, yeah right